recipes/libmemcached/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export CC=clang
+    export CXX=clang++
+
+    export MACOSX_VERSION_MIN=10.7
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
```

Comment:
All of this will be replaced by the `toolchain` activation as in this PR ( https://github.com/conda-forge/thrift-cpp-feedstock/pull/4 ) except for `-std=c++11`, which can be set outside of this `if` statement.